resources/views/auth/register.blade.php

```diff
@@ -1,77 +1,279 @@
 @extends('layouts.master')
-
 @section('body')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
+    @dump($errors)
```

Comment:
Might want to remove this (`@dump($errors)`), as users may be like "What's this thing??" :)